tests/gtest_sequence.cpp

```diff
@@ -240,11 +240,7 @@ TEST_F(SequenceTripleActionTest, TripleAction)
   using namespace BT;
   using namespace std::chrono;
 
-#ifdef WIN32
   const int margin_msec = 60;
-#else
-  const int margin_msec = 20;
```

Comment:
why not just increase the non win32 one?

Reply:
I think keeping the highest time margin is the most conservative solution. Should always work on both systems.